File: QBRTCDemo/Modules/Models/SVUser+QBUUser.m
//
//  SVUser+QBUUser.m
//  RTCDemo
//
//  Created by Anton Sokolchenko on 3/31/16.
//  Copyright © 2016 anton. All rights reserved.
//

#import "SVUser+QBUUser.h"

@implementation SVUser (QBUUser)

- (instancetype)initWithQBUUser:(QBUUser *)user {
	return [[SVUser alloc] initWithID:@(user.ID) login:user.login  fullName:user.fullName password:user.password tags:user.tags];
}

@end

File: QBRTCDemo/Modules/Models/QBUser+SVUser/SVUser+QBUUser.swift
//
//  SVUser+QBUUser.swift
//  QBRTCDemo
//
//  Created by Anton Sokolchenko on 3/13/19.
//  Copyright © 2019 Anton Sokolchenko. All rights reserved.
//

import Foundation
import Quickblox

extension SVUser {
	class func fromQBUUser(_ qbuuser: QBUUser) -> SVUser {
		let user = SVUser(ID: Int(qbuuser.id), login: qbuuser.login, fullName: qbuuser.fullName ?? "", password: qbuuser.password, tags: qbuuser.tags as? [String])
		return user
	}
}
